src/main/resources/network/brightspots/rcv/GuiTabulationPopup.fxml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ RCTab
+  ~ Copyright (c) 2017-2024 Bright Spots Developers.
+  ~
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  -->
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<Pane prefHeight="513.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1" fx:controller="network.brightspots.rcv.GuiTabulateController">
+   <children>
+      <VBox layoutX="14.0" layoutY="10.0" prefHeight="343.0" prefWidth="573.0" spacing="10.0">
+         <children>
+            <Text strokeType="OUTSIDE" strokeWidth="0.0" text="Configuration" wrappingWidth="176.1533203125">
+               <font>
+                  <Font name="System Bold" size="19.0" />
+               </font>
+            </Text>
+            <HBox prefHeight="45.0" prefWidth="567.0" spacing="30.0">
+               <children>
+                  <TextArea fx:id="filepath" editable="false" minWidth="200.0" prefHeight="39.0" prefWidth="500.0" style="-fx-background-color: red;" text="C:/filepath.json">
+                     <HBox.margin>
+                        <Insets left="15.0" />
+                     </HBox.margin>
+                  </TextArea>
+                  <Button fx:id="saveButton" minWidth="70.0" mnemonicParsing="false" onAction="#buttonSaveClicked" prefHeight="42.0" text="Save*" />
+                  <Button fx:id="tempSaveButton" minWidth="155.0" mnemonicParsing="false" onAction="#buttonTempSaveClicked" prefHeight="42.0" text="Save Temporary Config" />
```

Comment:
The "save temporary config" button is intended to be helpful for developers who don't want to constantly overwrite test files, then have to revert those changes in git. It is not shown to end users. It checks to see if the config version is `TEST`, and if so, it gives you the option.

Once you save a file, it overwrites the version number with the actual version, rather than `TEST`, and therefore your option to save a temporary file is removed. (This is the same functionality as today.)

I did make two changes based on this comment:
1. When trying to tabulate after File>New, "Save" was grayed out. I changed that. It now forces you to save an empty config.
2. The "Save" button's `*` wasn't getting correctly reset. Once removed via successful save, it stayed removed. I now add the `*` back in whenever a save is required.